pkg/machine/e2e/README.md

```diff
@@ -14,13 +14,15 @@ Note: you must not have any machines defined before running tests
 1. Open a powershell as admin
 1. $env:CONTAINERS_MACHINE_PROVIDER="hyperv"
 1. $env:MACHINE_IMAGE="https://fedorapeople.org/groups/podman/testing/hyperv/fedora-coreos-38.20230830.dev.0-hyperv.x86_64.vhdx.zip"

```

Comment:
@baude This still necessary?

Reply:
In reality, no; we can fix that down the road.